### PR TITLE
VN OKTA 399045 OIE SDKs and Samples page

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Added new “Identity Engine SDKs & Samples” page inside the Languages & SDKs section.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-399045](https://oktainc.atlassian.net/browse/OKTA-399045)

### CODE REVIEW

File: packages/@okta/vuepress-site/code/oie/index.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+title: Identity Engine SDKs & Samples
+---
+
+# Identity Engine SDKs & Samples
+
+<ApiLifecycle access="ie" />
+
+## What's new
+
+[Identity Engine](/docs/concepts/ie-intro/) introduces new identity management capabilities with greater flexibility and customization. To take advantage of these new features and for a better development experience, use the Identity Experience (IDX) SDKs to manage authentication in your apps. These new SDKs are recommended for orgs using the Identity Engine.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/oie/index.md:11]: IDX or IdX is an internal-use abbreviation only AFAIK??
Looks like everywhere else but just this one instance you use just "Identity Engine" not "the Identity Engine". So, just fixing this one.

```suggestion
[Identity Engine](/docs/concepts/ie-intro/) introduces new identity management capabilities with greater flexibility and customization. To take advantage of these new features and for a better development experience, use the Identity Experience SDKs to manage authentication in your apps. These new SDKs are recommended for orgs using Identity Engine.
```

File: packages/@okta/vuepress-site/code/oie/index.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+<!-- Identity Engine introduces new capabilities and design patterns. To take advantage of these new features and provide a better development experience, new Okta Identity Experience (IDX) SDKs are available and recommended for orgs using Identity Engine. -->
+
+Experiment with sample apps that showcase Identity Engine using the following [authentication approaches](/docs/concepts/redirect-vs-embedded/):
+<!-- We also have new sample apps for you to experiment with Identity Engine authentication, for multiple authentication approaches: -->
+
+* **Redirect**: These sample apps demonstrate how to redirect users to an Okta-hosted sign-in page, and then receive users redirected back from Okta after sign in. This approach is recommended for most developers, as it is easier to build and maintain.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/oie/index.md:17]: In the Best Practices for InfoDev Styles section of the style guide, we talk about not using sign-in as a noun. 

```suggestion
* **Redirect**: These sample apps demonstrate how to redirect users to an Okta-hosted sign-in page, and then receive users redirected back from Okta after users sign in. This approach is recommended for most developers, as it is easier to build and maintain.
```

**Review by susanharper-okta — COMMENTED** [review]
what is complete so far looks good from a Tech Writer perspective....the decision on the URLs for redirect and embedded are above my pay grade, haha.

File: packages/@okta/vuepress-site/code/oie/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/code/oie/index.md:11]: Thanks. Depending on what PM says, I might have to remove all IDX references in names of the GitHub links.

**Review by alexdahl-okta — COMMENTED** [review]
Thanks, this is looking great!

File: packages/@okta/vuepress-site/code/oie/index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+##### iOS
+
+* [Swift IDX SDK](https://github.com/okta/okta-idx-swift) 
+* [iOS sample app](https://github.com/okta/samples-ios)
+	- see [Browser Sign In](https://github.com/okta/samples-ios/tree/master/browser-sign-in) for redirect sample configuration
+	- see [Custom Sign In](https://github.com/okta/samples-ios/tree/master/custom-sign-in) for embedded sample configuration
+* iOS IDX SDK (??? is there another iOS SDK?)
+* iOS Redirect/Embedded Sample App (??? sample app downloads are the same for redirect/embedded)
+* More iOS resources (???)
```
[the remark this answers is not in the document]
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/code/oie/index.md]: Will do. Thanks!

File: packages/@okta/vuepress-site/code/oie/index.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+* [.NET IDX SDK](https://github.com/okta/okta-idx-dotnet)
+* [ASP.NET MVC 4.x embedded-IDX sample app](https://github.com/okta/okta-idx-dotnet/tree/master/samples/samples-aspnet)
+* [ASP.NET MVC 4.x sample app](https://github.com/okta/samples-aspnet) (??? Classic engine?)
+* [ASP.NET Core 2.x and 3.x sample app](https://github.com/okta/samples-aspnetcore)
+* [ASP.NET Web Forms redirect sample app](https://github.com/okta/samples-aspnet-webforms)
+	- see [Okta-Hosted Login](https://github.com/okta/samples-aspnet-webforms/tree/master/okta-hosted-login) for redirect sample configuration
+	- see [Self-Hosted Login](https://github.com/okta/samples-aspnet-webforms/tree/master/self-hosted-login) for embedded sample configuration (???)
+* More .NET resources (???)
+
+##### Other
```
[the remark this answers is not in the document]
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/code/oie/index.md]: Moved sample apps to the proper technology headings. Thanks.

File: packages/@okta/vuepress-site/code/oie/index.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+##### iOS
+
+* [Swift Identity Engine SDK](https://github.com/okta/okta-idx-swift)
+* [iOS embedded authentication with SDK sample app](https://github.com/okta/okta-idx-swift/tree/feat-dynamic-auth/Samples/EmbeddedAuthWithSDKs)
+* [iOS redirect sample app](https://github.com/okta/samples-ios) &mdash; see [Browser Sign In](https://github.com/okta/samples-ios/tree/master/browser-sign-in) for redirect configuration
+
+##### React Native
+
+* [React Native redirect sample app](https://github.com/okta/samples-js-react-native) &mdash; see [Browser Sign In](https://github.com/okta/samples-js-react-native/tree/master/browser-sign-in) for redirect configuration
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/oie/index.md:40]: should this be split out by React Native iOS and React Native Android for redirect authentication?

File: packages/@okta/vuepress-site/code/oie/index.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+Orgs using Identity Engine can also continue to use current SDKs and tools. You can find those SDKs for your stack from the [Languages & SDKs overview](/code/).
+
+## Repository
+
+### Mobile
+
+##### Android
+
+<!--* [Android Identity Engine SDK](https://github.com/okta/okta-idx-android)-->
+* [Android redirect sample app](https://github.com/okta/samples-android) &mdash; see [Browser Sign In](https://github.com/okta/samples-android/tree/master/browser-sign-in) for redirect configuration
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/oie/index.md:30]: global comment - this should be "redirect authentication sample app". We should be referring to redirect authentication just like we do for embedded authentication. Thanks!

File: packages/@okta/vuepress-site/code/oie/index.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+##### .NET
+
+* [.NET Identity Engine SDK](https://github.com/okta/okta-idx-dotnet)
+* [ASP.NET MVC embedded authentication with SDK sample app](https://github.com/okta/okta-idx-dotnet/tree/master/samples/samples-aspnet/embedded-auth-with-sdk)
+
+* [ASP.NET MVC redirect sample app](https://github.com/okta/samples-aspnet) &mdash;  see [Okta-Hosted Login](https://github.com/okta/samples-aspnet/tree/master/okta-hosted-login) for redirect configuration
+* [ASP.NET Web Forms redirect sample app](https://github.com/okta/samples-aspnet-webforms) &mdash;  see [Okta-Hosted Login](https://github.com/okta/samples-aspnet-webforms/tree/master/okta-hosted-login) for redirect configuration
+* [Blazor redirect sample app](https://github.com/okta/samples-blazor) &mdash;  see [Blazor Server-Side Okta-Hosted Login](https://github.com/okta/samples-blazor/tree/master/server-side/okta-hosted-login) for redirect configuration
+* [ASP.NET Core 2.x and 3.x redirect sample app](https://github.com/okta/samples-aspnetcore) &mdash;  see [Okta-Hosted Login](https://github.com/okta/samples-aspnetcore/tree/master/samples-aspnetcore-2x/okta-hosted-login) for redirect configuration
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/oie/index.md:73]: this should only reference the 3.x asp.net core - 2.x is no longer supported. Also change the link to the 3x directory for this one.

**Review by susanharper-okta — COMMENTED** [review]
just a couple questions.

File: packages/@okta/vuepress-site/code/oie/index.md (hunk 7 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/code/oie/index.md]: thanks!

File: packages/@okta/vuepress-site/code/oie/index.md (hunk 6 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/code/oie/index.md]: global update done. thanks!

File: packages/@okta/vuepress-site/code/oie/index.md (hunk 5 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/code/oie/index.md]: I was going to at first, but the sample app download is the same for both iOS and Android, and the "Browser Sign In" instructions contain both iOS and Android content.

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚢

**Review by susanharper-okta — APPROVED** [review]
did a spot check locally of some of the links - looks good to me!

**Pull request merged**